### ISSUE TITLE
Not really an issue, just a question: How to instantiate the class?

### ISSUE THREAD

**Issue by ultrablue** [roufamatic/todo-txt-js#1]
Hi:

I really like the way you've approached ToDo.txt in this class. However, I'm having a hard time instantiating it as a new object with no data. The only thing I've gotten to work is this:

var todos = TodoTxt.parseFile("");

I'm fairly green with OO JavaScript -- am I missing something?

Thanks!
Greg

**Comment by roufamatic** [comment]
Hi Greg, thank you for your comment!

This project came about to support my Chrome extension, Pay the Piper
https://chrome.google.com/webstore/detail/pay-the-piper/iiepcdnmdcdplpmckjlaefajphcbelcm?hl=en-US
.
As such, what you see is pretty much just what I needed to complete my
project. I started with an existing todo.txt file, so my first goal was to
create something that could parse it. The zero case for me was to do
exactly what you describe: parse the empty string. Makes perfect sense to
me! But totally dumb for a new user with different requirements. If you'd
like to add a new method called TodoTxt.create() that just calls
parseFile("") under the hood I think that would be quite sensible. (But
promise me you'll add a test too!)

Also, since you're new to OO JS, I'll put in a quick recommendation: read
Douglas Crockford's book "Javascript: The Good Parts". He describes three
different ways to write OO Javascript: classical, prototypal, and
functional. My style follows his recommendations of abandoning "new" and
the other Java-like crutches and focusing on a purely functional
implementation. I cannot recommend this book enough, it will make you a
better JS programmer.

Good luck!
Mike
